pkgs/development/python-modules/pyqt/6.x.nix

```diff
@@ -25,17 +26,15 @@
 
 buildPythonPackage rec {
   pname = "pyqt6";
-  version = "6.7.0.dev2404081550";
+  version = "6.8.0.dev2410141303";
   format = "pyproject";
 
   disabled = pythonOlder "3.6";
 
+  # This is dangerous, how can we get web archive to archive the URL?
```

Comment:
I suppose web.archive.org is still in read-only operation following the recent DDoS attack.

Reply:
I assume this is a blocker for us since we can't trust this version to stay up. Do we have any other workarounds to un-break master?

Reply:
We could just do the same thing as Arch does, merge as is and open a new PR when it breaks. But that feels wrong since it's a false security.

Reply:
The source would be cached by hydra. And adding archive urls in the future wouldn't cause rebuild. Good enough for us now.